src/controllers/expenses.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const { ExpenseModel } = require('../models/expenses');
+const { UserModel } = require('../models/users');
+
+const getAllExpenses = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const expenses = ExpenseModel.getExpenses(userId, categories, from, to);
+
+  res.send(expenses);
+};
+
+const addNewExpense = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  if (!userId || !spentAt || !title || !amount || !category) {
+    res.status(400).send('Bad request');
+
+    return;
+  }
+
+  const newExpense = ExpenseModel.addExpense({
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  });
+
+  const user = UserModel.getUser(+userId);
+
+  if (!newExpense || !user) {
+    res.status(400).send('Bad request');
```

Comment:
The error message 'Bad request' is too generic. Consider providing more specific feedback, such as 'Invalid user ID or expense data' to help with debugging.